src/clients/FlexbaseClient.Banking.ts

```diff
@@ -43,6 +54,30 @@ export class FlexbaseClientBanking extends FlexbaseClientBase {
         params.isPdf = true;
       }
 
+      if (options?.fromDate) {
```

Comment:
This huge block is sorta repetitive. Might be good to clean it up with a for...in loop

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in